MediaManager/Media/MediaExtractorBase.cs

```diff
@@ -14,6 +17,39 @@ public virtual Task<IMediaItem> CreateMediaItem(string url)
             return UpdateMediaItem(mediaItem);
         }
 
+        public virtual async Task<IMediaItem> CreateMediaItem(string resourceName, Assembly assembly)
+        {
+            string path = null;
```

Comment:
if(assembly is null)
AppDomain.GetEntry()

Reply:
Added `TryFindAssembly`.